ha/core/system_health/get_system_attributes.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+import ast
+
+from ha  import const
+from ha.core.system_health.system_health import SystemHealth, SystemHealthManager
+from ha.core.config.config_manager import ConfigManager
+from ha.core.error import ClusterManagerError
+
+class NodeParams:
+    @staticmethod
+    def get_node_map(node_name: str) -> dict:
```

Comment:
get_system_attributes.py

Why this class needed
separate work
can we combine it in other class

Reply:
We need to get the node map in multiple places in our code.
So separate function is created.
There are also other data points that are needed : ex:  get node_id given a node_name or vice -versa.

All those can be added in this file . So created separate file .

In future there can be parameters specific to other entities like enclosure / storageset etc. So class is created

Reply:
we can also access node map with system health

key = self._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=node_id)

Reply:
Moved the function to system_health.py